### PR TITLE
Design style avatar uploadbtn

### PR DESCRIPTION
**Pull request by sulaimancode**
Uploading avatar filed button looks pretty now.

### CODE REVIEW

**Review by ffasolin — APPROVED** [review]
Looks good to me!

File: app/assets/stylesheets/forms.scss [hunk 1]
```diff
@@ -84,3 +84,38 @@ input {
 h2 {
   text-align: center;
 }
+
+.fileUpload {
```
Comment (neoeno) [inline app/assets/stylesheets/forms.scss]: Would `.file-upload` work here? It's more according to the class naming conventions.

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -14,10 +14,12 @@
     <div class='password-field info'>
       <%= form.password_field :password, placeholder: "Password..." %>
     </div>
-
-    <div class='text-field'>
-      <%= form.label :avatar %>
-      <%= form.file_field :avatar %>
+    <div class="choose-file">
+    <input id="uploadFile" placeholder="Avatar File" disabled="disabled" />
+  </div>
```
Comment (neoeno) [inline app/views/users/new.html.erb]: Indentation

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -12,3 +12,16 @@
...
+  if (document.readyState != 'loading'){
+    fn();
+  } else {
+    document.addEventListener('DOMContentLoaded', fn);
+  }
+}
+
+ready(function() {
+  document.getElementById("user_avatar").onchange = function () {
+    document.getElementById("uploadFile").value = this.value;
```
Comment (neoeno) [inline app/assets/javascripts/application.js]: Underscores are convention for IDs: `upload_file`

Comment (neoeno) [reply app/assets/javascripts/application.js]: Ha — this UI has a real retro feel. I like it! I guess you noticed the problem that there's no user feedback when they add an avatar, and that's why the text box is there?

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -14,10 +14,12 @@
     <div class='password-field info'>
       <%= form.password_field :password, placeholder: "Password..." %>
     </div>
-
-    <div class='text-field'>
-      <%= form.label :avatar %>
-      <%= form.file_field :avatar %>
+    <div class="choose-file">
+    <input id="uploadFile" placeholder="Avatar File" disabled="disabled" />
```
Comment (neoeno) [inline app/views/users/new.html.erb]: This `input` tag should have a `type` attribute I think.

File: app/assets/stylesheets/forms.scss [hunk 5]
```diff
@@ -84,3 +84,38 @@ input {
...
+    filter: alpha(opacity=0);
+}
+
+.choose-file {
+  float: left;
+}
+
+.fileUpload:hover {
+  background-color: rgba(255, 255, 255, 0.1);
+}
```
Comment (neoeno) [inline app/assets/stylesheets/forms.scss]: There's some inconsistent indentation in this file.

**Review by neoeno — CHANGES_REQUESTED** [review]
We can get rid of the DS_Store files I think, they don't serve any purpose for us.

Nice work!

File: app/views/users/new.html.erb (hunk 4 above)
Comment (sulaimancode) [reply app/views/users/new.html.erb]: I think that input tag is there just to show user feedback, and it's value is changed to the name of whatever file the user selects.

File: app/assets/javascripts/application.js (hunk 3 above)
Comment (sulaimancode) [reply app/assets/javascripts/application.js]: Yes, that is the purpose of the text box. still need to change id to follow conventions

**Comment by sulaimancode** [comment]
Made all the requested changes

**Review by neoeno — APPROVED** [review]
Nice work!

**Pull request merged**